protocol/avdl/chat1/common.avdl

```diff
@@ -34,6 +34,7 @@ protocol common {
   }
 
   record RateLimit {
+    string name;
```

Comment:
Would it be helpful to make this into a int backed type to safe some bandwidth and potentially avoid typo caused bugs?